staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/start.go

```diff
@@ -98,7 +99,7 @@ func (o CustomResourceDefinitionsServerOptions) Config() (*apiserver.Config, err
 	}
 
 	serverConfig := genericapiserver.NewRecommendedConfig(apiserver.Codecs)
-	if err := o.RecommendedOptions.ApplyTo(serverConfig); err != nil {
+	if err := o.RecommendedOptions.ApplyTo(serverConfig, runtime.NewScheme(), nil); err != nil {
```

Comment:
the scheme must be the one of the apiserver at hand, i.e. the one where the types are registered that are sent to the admission plugins. Something like that also has to be in the godoc of ApplyTo.

Background: the webhook admission plugin uses the scheme to convert the types to an external version that a webhook understands.

Reply:
ok, I have to learn about scheme, actually I am not familiar with it.

Reply:
know a little about it. My usage is wrong here.